app/controllers/teams/team_memberships_controller.rb

```diff
@@ -0,0 +1,17 @@
+class Teams::TeamMembershipsController < ApplicationController
+  before_action :authenticate_user!
+
+  def destroy
+    team_membership = teams_managed_members.find(params[:id])
+
+    team_membership.destroy
+
+    redirect_to teams_team_memberships_path(team_membership.team)
+  end
+
+  private
+
+  def teams_managed_members
```

Comment:
I don't think this needs to be it's own method. It just adds indirection. I think a single statement is more readable: 
```
TeamMembership.where(team_id: current_user.teams_managed.select(:id)).
               find(params[:id])
```